delayed.py

```diff
@@ -90,8 +105,15 @@ def host_info_obtainer(conn, vc):
 
     process_actions = True
     while process_actions:
+
+        with data.Connection.use('conn2') as conn:
+            try:
+                host_info_obtainer(conn, vc)
+            except Exception:
+                Settings.raven.captureException(exc_info=True)
+                logger.error('Exception while processing request: ', exc_info=True)
```

Comment:
I would maybe propose:
```suggestion
                logger.error('Could not obtain host information: ', exc_info=True)
```

Reply:
OK